### PR TITLE
Update docs to reflect billing UI changes

### PR DESCRIPTION
**Pull request by usha-mandya**
Signed-off-by: Usha Mandya <usha.mandya@docker.com>

Update billing docs to ensure they reflect the Billing frontend updates.

### CODE REVIEW

**Comment by usha-mandya** [comment]
https://deploy-preview-12585--docsdocker.netlify.app/docker-hub/billing/upgrade/

File: docker-hub/billing/upgrade.md [hunk 1]
```diff
@@ -23,19 +23,23 @@ To upgrade to a Pro plan:
 
 1. Log into your [Docker Hub](https://hub.docker.com){: target="_blank" rel="noopener" class="_"} account.
 
-2. Open the drop-down menu next to your username in the top-right corner and select **Billing**. The Billing page displays billing information about your existing user account.
+2. Open the drop-down menu next to your username in the top-right corner and select **Billing**. The **Billing Details** page displays billing information about your existing user account.
 
-3. Click **Change plan** to choose a new plan.
+3. Ensure you are updating the billing details  your personal namespace.
```
Comment (mark-dr) [inline docker-hub/billing/upgrade.md:28]:
```suggestion
3. Ensure you are updating the billing details of your personal namespace.
```

File: docker-hub/billing/downgrade.md [hunk 2]
```diff
@@ -24,21 +24,23 @@ To downgrade from a Pro plan to a Free plan:
 
 1. Log into your [Docker Hub](https://hub.docker.com){: target="_blank" rel="noopener" class="_"} account.
 
-2. Open the drop-down menu next to your username in the top-right corner and select **Billing**. The Billing page displays billing information about your existing user account.
+2. Open the drop-down menu next to your username in the top-right corner and select **Billing**. The **Billing Details** page displays billing information about your existing user account.
 
-3. Click **Change plan** to downgrade your existing plan.
+3. Ensure you are updating the billing details of your personal namespace.
```
Comment (mark-dr) [inline docker-hub/billing/downgrade.md:29]: I'd be more explicit here, personally - something like

```suggestion
3. Ensure your personal namespace is selected in the list on the left of the page.
```

File: docker-hub/billing/upgrade.md [hunk 3]
```diff
@@ -77,8 +81,9 @@ To upgrade from a monthly Team plan to an annual Team plan:
 To upgrade from a monthly Pro plan to an annual Pro plan:
 
 1. Log into your [Docker Hub](https://hub.docker.com){: target="_blank" rel="noopener" class="_"} account.
-2. Open the drop-down menu next to your username in the top-right corner and select **Billing**. The Billing page displays billing information about your existing user account.
-3. Click **switch to annual billing**.
-4. Review the information displayed on the **Change to an Annual Plan** page and click **Accept Terms and Purchase** to confirm.
+2. Open the drop-down menu next to your username in the top-right corner and select **Billing**. The **Billing Details** page displays billing information about your existing user account.
```
Comment (mark-dr) [inline docker-hub/billing/upgrade.md:84]: It feels abrupt to jump from talking about **Billing** to **Billing Details**; I might not understand the implicit connection - that clicking the former will take you to the latter.

```suggestion
2. Open the drop-down menu next to your username in the top-right corner and select **Billing** to visit the **Billing Details** page, which displays billing information about your existing user account.
```

File: docker-hub/billing/downgrade.md (hunk 2 above)
Comment (usha-mandya) [reply docker-hub/billing/downgrade.md]: Updated

File: docker-hub/billing/upgrade.md (hunk 3 above)
Comment (usha-mandya) [reply docker-hub/billing/upgrade.md]: Updated

File: docker-hub/billing/upgrade.md (hunk 1 above)
Comment (usha-mandya) [reply docker-hub/billing/upgrade.md]: Updated

**Review by FranchommeE — COMMENTED** [review]
@usha-mandya Looks good to me for Pro plans related changes.
I'll talk to @mark-dr to understand the release plan. If we release the new Billing page "not consolidated" first, these docs updates are fine.
However, the end goal is to let the user switch between personal / org namespace and view the billing details on the same page. So we would need to also update the Org/Team related paragraphs in the docs too.

**Comment by mark-dr** [comment]
@FranchommeE, the initial plan is that the consolidated view will only appear when the user selects _Billing_ from the top-right nav dropdown. It won't appear when viewing Billing for specific orgs, so the existing descriptions of the journeys still work fine.

**Review by mark-dr — DISMISSED** [review]

**Review by FranchommeE — APPROVED** [review]
LGTM

**Review by mark-dr — APPROVED** [review]

**Comment by usha-mandya** [comment]
Merging to master as the updates are ready to go live.

**Pull request merged**